services/thermal_mgr/thermal_mgr.c

```diff
@@ -41,20 +41,57 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 }
 
+#define MAX_MS_WAIT 20
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  xQueueSend(thermalMgrQueueHandle, (void *)event, portTICK_PERIOD_MS*MAX_MS_WAIT);
+  return ERR_CODE_SUCCESS;
+}
 
+error_code_t thermalMgrSendEventISR(thermal_mgr_event_t *event) {
+  /* Send an event to the thermal manager queue */
+  xQueueSendFromISR(thermalMgrQueueHandle, (void *)event, NULL);
```

Comment:
either check the return type on this or make this function void

Reply:
Checked for return type and return ERR_CODE_QUEUE_FULL